updated LRAR page (#37)
ROvACC/training-docs: docs/source/airports/arad-tma/LRAR.rst

LRAR - Arad International
=========================

General
"""""""

> Arad Airport it is one of those airports in Romania where there is not much international traffic and where not many airlines operate. However, the airport is an important point in air navigation service because it is a provider for control in TMA Arad and soon for the first Virtual Tower in the country.

Reviewer: ```suggestion
Arad Airport, while being an international one, is not an airport regularly operated by any of the commercial airlines. However, the airport is an important point in air navigation service as it is a provider for control in TMA Arad and soon for the first Virtual Tower in the country.
```

| Prefered config | ARR          | DEP          |
+-----------------+--------------+--------------+
| Deicing         | on parking position         |
+-----------------+--------------+--------------+

Clearance
"""""""""

Both RNAV and conventional SIDs are available, but RNAV are highly recommended. Non standard departure shall be coordinated with APP unit.

Taxi
""""

Departing and arrived traffic shall be instructed to follow standard taxi routes available on AIP LR AD 2.1-14.

Stand assignments
"""""""""""""""""

It is generally at the discretion of the ATC.

+-------+--------+-------------------+
| Apron | Stands | Procedure         |
+-------+--------+-------------------+
| 1     | 3-5    | self maneuvering  |
+-------+--------+-------------------+
| 2     | 1-2    | self maneuvering  |
+-------+--------+-------------------+

LVP
"""

During LVP, aircrafts shall be instructed to follow standard taxi routes with Follow Me guidance.

Traffic regulation
""""""""""""""""""

CTR Arad is classified as Class C Airspace where both IFR and VFR are allowed to enter.

CTR Cluj lower limit is the ground level and the upper limit is 2500 ft AMSL (by QNH).

> VFR traffic circuits are allowed at 1500 ft AMSL (by QNH). VFR traffic could be cleared to climb in TMA only by APP unit. A coordination between TWR and APP units shall be in force.

Reviewer: ```suggestion
VFR traffic circuits are allowed at max 1500 ft AMSL (by QNH). VFR traffic could be cleared to climb in TMA only by the APP unit. Coordination between TWR and APP units shall be in force.
```
